### DISCUSSION TITLE
Is it possible to connect a 16x2 LCD Screen?

### DISCUSSION THREAD

**Discussion by jujuKUHL** [gpiozero/gpiozero#1125] [Q&A] [answered]
Hey, i‘m pretty new to Raspberry Pi 5 and Pi in general. We have a School Project with Pi and Python. I want to connect a 16x2 LCD Screen (no I2C) with the gpiozero Library, but in the Documentation I don‘t find anything that this is even supported.

Does someone already did this and could give me a hint how to do that and to write an example text on the screen?

**Comment by lurch** [comment, marked as answer]
Gpiozero doesn't have built-in support for 16x2 LCD screens, so you might be better off looking for some other library.

**Reply by jujuKUHL** [reply]
Thanks for the answer. Do you know which library I could use for that?

**Reply by lurch** [reply]
Not off the top of my head, but this should point you in the right direction https://www.google.com/search?q=raspberry+pi+python+hd44780

**Comment by Nickleaton** [comment]
Check out Seeed [3 e's] or Adafruit.

N.

On Thu, 22 Feb 2024 at 16:06, jujuKUHL ***@***.***> wrote:

> Thanks for the answer. Do you know which library I could use for that?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gpiozero/gpiozero/discussions/1125#discussioncomment-8558819>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAF7NEEELZVAXLWWUTSB5C3YU5UJVAVCNFSM6AAAAABDUWRPXGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKNJYHAYTS>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Nick
